app/views/grade_entry_forms/_form.html.erb

```diff
@@ -87,5 +87,10 @@
     <%= f.submit (f.object.new_record? ? t('helpers.submit.create', model: GradeEntryForm.model_name.human) :
                                          t('helpers.submit.update', model: GradeEntryForm.model_name.human)),
                  data: { disable_with: t('working') } %>
+    <% if @grade_entry_form.id != nil and @grade_entry_form.count_non_nil == 0 %>
```

Comment:
Touch base with @mimischly7 about both how the button should be styled and using the `disabled` attribute